<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jmcStopWidget</class>
 <widget class="QWidget" name="jmcStopWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="cSuperBtn" name="Execute">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;</string>
   </property>
   <property name="text">
    <string>停止触发</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;

</string>
   </property>
   <property name="text">
    <string>减速度</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="cSuperBtn" name="fDec">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;

</string>
   </property>
   <property name="text">
    <string>减减速度</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="cSuperBtn" name="fJerk">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;

</string>
   </property>
   <property name="text">
    <string>错误码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="cSuperBtn" name="errorCode">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="cSuperBtn" name="busy">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;</string>
   </property>
   <property name="text">
    <string>busy</string>
   </property>
  </widget>
  <widget class="cSuperBtn" name="done">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;</string>
   </property>
   <property name="text">
    <string>done</string>
   </property>
  </widget>
  <widget class="cSuperBtn" name="error">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;</string>
   </property>
   <property name="text">
    <string>error</string>
   </property>
  </widget>
  <widget class="cSuperBtn" name="aborted">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;</string>
   </property>
   <property name="text">
    <string>aborted</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cSuperBtn</class>
   <extends>QPushButton</extends>
   <header>include/cwidget/cSuperBtn.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
